pkgs/by-name/rq/rqbit/package.nix

```
@@ -45,15 +45,11 @@ in
 rustPlatform.buildRustPackage {
   inherit pname version src;
 
-  cargoHash = "sha256-esDUzzVm5J8fKftBfk5StJzN1YzLa1p0t7BsoxzrowI=";
+  cargoHash = "sha256-cJRv+b780daYeiAFnbhYoYNwOy9GL8x9Ve6XxG95QpU=";
```

```suggestion
  useFetchCargoVendor = true;
  cargoHash = "";
```
We are moving to our new cargo fetcher nixpkgs-wide, please regenerate the hash.
LGTM otherwise, though I did not yet test it.